CHANGELOG.md

```diff
@@ -5,6 +5,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- A [`manifest.yml`](./manifest.yml) has been added, allowing for
```

Comment:
Ruh-Roh, CodeClimate is complaining about a few markdown issues that show up in all of our CHANGELOG.md files: Headers and lists should be surrounded by blank lines. We'll probably need to clean this up at some point.

Reply:
parse-a-changelog fails if we do that :/ 
can we add an exception for this file?

Reply:
I can override codeclimate for this PR - let me know when it's approved to merge, and I will override the CC failure this one time and merge it.

Reply:
`parse-a-changelog` might be being a bit too restrictive there. Maybe we should just allow those blank lines, to avoid having to need overriding codeclimate again